FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -694,27 +706,27 @@
 				<owl:hasValue rdf:resource="&fibo-fbc-fct-usjrga;FederalReserveRegulatoryAgencyAndCentralBank"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>Federal Reserve System (FRS) member bank</rdfs:label>
+		<rdfs:label>FRS member bank</rdfs:label>
 		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies/"/>
-		<skos:definition>a Federal Reserve member bank</skos:definition>
+		<skos:definition>financial institution that is a member of the Federal Reserve System (FRS)</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://federalreserve.gov/</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fbc-fct-usjrga;FRSNonMemberBank">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-fct-fse;FinancialInstitution"/>
-		<rdfs:label>Federal Reserve System (FRS) non-member bank</rdfs:label>
+		<rdfs:label>FRS non-member bank</rdfs:label>
```

Comment:
If it's a "bank" why does it not subclass fibo-fbc-fct-fse;Bank? There is a mismatch between the name and the superclass.

Reply:
See above - it's due to the usage of the term 'bank', which is broader than our definition.

Reply:
Created #890